README.md

```diff
@@ -12,8 +12,13 @@ osd-network-verifier can be used prior to the installation of osd/rosa clusters
 
 * it creates an instance in the target vpc/subnet and wait till the instance gets ready
 * when the instance is ready, an `userdata` script is run in the instance. The `userdata` mainly performs 2 steps, it
-* installs appropriate packages, primarily docker
-* runs the validation image against the vpc/subnet as containerized form of <https://github.com/openshift/osd-network-verifier/tree/main/build>
+    * installs appropriate packages, primarily docker
+    * runs the validation image against the vpc/subnet as containerized form of <https://github.com/openshift/osd-network-verifier/tree/main/build>
+        * the image is available at: https://quay.io/repository/app-sre/osd-network-verifier?tag=latest&tab=tags
+        * it can be run either locally or on an instance on the target vpc via: 
```

Comment:
might be obvious but it could be worth rephrasing (or specifying)`run locally`.
Since running the `docker run` locally will not validate against the vpc/subnet.